packages/synapse-react-client/src/style/components/entity_finder/_details-view.scss

```diff
@@ -36,7 +36,7 @@ $row-hovered-background-color: unset;
 .EntityFinderDetailsViewPlaceholder {
   @include SrcMixins.placeholder;
   .SearchPlaceholderImage {
-    width: 150px;
+    height: 150px;
```

Comment:
Adjusting the dimensions so that the new icon is similarly sized to the previous icon in the EntityFinder